compiler/p/codegen/OMRLinkage.cpp

```diff
@@ -685,16 +600,18 @@ TR::Instruction *OMR::Power::Linkage::flushArguments(TR::Instruction *cursor)
 
                if (paramCursor->getDataType() == TR::Float)
                   {
-                  op = TR::InstOpCode::stfs; length = 4;
+                  op = flush ? TR::InstOpCode::stfs : op = TR::InstOpCode::lfs; length = 4;
```

Comment:
The extra `op =` here (and in a similar line below) is redundant and confusing. It would be better to do:

```suggestion
                  op = flush ? TR::InstOpCode::stfs : TR::InstOpCode::lfs; length = 4;
```

Reply:
Fixed in the new commit 54be265